docs/ops.html

```diff
@@ -3898,6 +3904,12 @@ <h3>Migrating brokers to KRaft</h3>
 # Remove ZooKeeper client configuration
 # zookeeper.connect=localhost:2181
 
+# Remove the inter broker listener in brokers to allow KRaft controller send RPCs to brokers
+# inter.broker.listener.name=PLAINTEXT
+
+# Maps listener names to security protocols. Please add the inter broker listener protocol mapping
+# listener.security.protocol.map=PLAINTEXT:PLAINTEXT,CONTROLLER:PLAINTEXT
+
```

Comment:
I think this change should be removed. This configuration should not be removed and it's already set above online 3896.

Reply:
Had another look, I think I was wrong. This section is to remove the configs in the broker side, but what we want to add is in the KRaft controller config. Removed this change.